problems/1.types+operators.js

```diff
@@ -1,21 +1,46 @@
 /** Takes in an value and returns if it is a number
-* @param {any} val - The input value
-* @returns {boolean} - Is val a number
-*/
-function isNumber() {
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val a number
+ */
+function isNumber(a, b) {
```

Comment:
function should only take in 1 argument. use typeof to check if that arg is equal to 'number'

Reply:
this inputs are so valuable I will keep these notes.